spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayProperties.java

```diff
@@ -355,6 +367,12 @@
 	 */
 	private List<String> vaultSecrets;
 
+	/**
+	 * Whether to fail if a location specified in the flyway.locations option doesn't
+	 * exist.
+	 */
+	private boolean failOnMissingLocations = false;
```

Comment:
```suggestion
	private boolean failOnMissingLocations;
```
Because boolean's default value is false.